### PR TITLE
15-view-by-type :: View Product Listings by Product Type

### LINKED ISSUE #15: Generate Product List (by Type) View
### Feature Description

Create a view that will list all products of a particular type.
### Requirement to Fulfill

> Refer to the [project requirements](https://github.com/TeamCharles/bangazon-inc/blob/master/projects/INITIAL_BANGAZON_SITE.md#requirements) before filling out this section.
1. Provide a view to show all products that are of a particular product type.
2. List all products, with the name text as a hyperlink to the detail view.

### PR DESCRIPTION
**Pull request by MattKraatz**
## Status

**READY**
## Documentation

[X] I have fully documented features in this PR. Please put an `X` in the box to confirm.
## Migrations

NO
## Description

Created a view for products of a particular Product Type.
## Related PRs

This builds on top of PR #68 
## Resolves Issue Number

Resolves #15 
## Deploy Notes

N/A
## Steps to Test or Reproduce
- Navigate to localhost:5000
- Click on any product type
- Verify that all products of that type are listed

``` sh
git pull --prune
git checkout 15-view-by-type
dotnet run
```
## Impacted Areas in Application

List general components of the application that this PR will affect:
- Product view

### CODE REVIEW

**Comment by daynewright** [comment]
LGTM

File: Views/ProductTypes/List.cshtml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+<p>
+    <a asp-action="Create">Sell Something</a>
+</p>
+
+@foreach (Product item in Model)
+{
+    <div class="row">
+        <div class="col-md-3">@Html.DisplayFor(modelItem => item.Name)</div>
+        <div class="col-md-3">@Html.DisplayFor(modelItem => item.Price)</div>
```
Comment (matthamil) [inline Views/ProductTypes/List.cshtml]: Could add a `$` in front of `@Html.DisplayFor(modelItem => item.Price)` to make it more clear that this number being displayed is the price.

**Review by matthamil — DISMISSED** [review]

**Comment by Anulfo** [comment]
LGTM

**Review by daynewright — APPROVED** [review]
LGTM

**Pull request merged**